### ISSUE TITLE
Automatically generated default scenario?

### ISSUE THREAD

**Issue by sfischer13** [huginn/huginn#2196]
I would like to export all of agent configurations at once. This should be possible if I added all my agents to a single scenario but then I would have to maintain that scenario.
It would it be nice to have an automatically generated default scenario that contains all agents. Or is there another (user friendly, non-admin) way to save all of a user's agents?

**Comment by dsander** [comment]
I think having all Agents in a default scenario would clutter the UI a bit, we display the Agents scenarios in a few places which would get repetitive.

Do you intend to use this a kind of backup?

**Comment by sfischer13** [comment]
You're right about cluttering the interface. I was trying to find a simple way of saving a user's agents without touching the database as I don't really care about losing my agents' history. Maybe that's a personal use case. On the other hand, a button "Save my agent settings" could be added to the account page. But now I'm just thinking out loud ... Anyway, thanks for your patient feedback.

**Comment by dsander** [comment]
Adding a global backup function should not be too complicated, what is more challenging is the import. Simply removing all Agents and Credentials before importing could be an option to avoid having to deal with merging Agent configuration like we do in the Scenario import.

**Comment by sfischer13** [comment]
That's the kind of scenario I had in mind. If I (as a user) mess up my agents, how can I restore a previous configuration.
